test/pjrt/test_experimental_pjrt_tpu.py

```diff
@@ -160,6 +168,9 @@ def test_global_ordinal(self, ordinal_func):
 
   @parameterized.named_parameters(('xla_model', xm.get_local_ordinal),
                                   ('pjrt', pjrt.local_ordinal))
+  @absltest.skipIf(
+      tpu.version() <= 2,
+      'This test is not currently supported on v2 TPUVMs or earlier.')
   def test_local_ordinal(self, ordinal_func):
     accelerator_num_devices = {
         'v3-8': 8,
```

Comment:
This version of the test should also work on v2, but it would be clunky to specify `v2-8` and `v3-8` separately for each of these tests. Can you file a follow-up bug to add a better utility to specify the correct output for these tests?

Reply:
Created https://github.com/pytorch/xla/issues/4383